base_custom_filter/tests/test_filters.py

```diff
@@ -4,8 +4,21 @@
 @tagged("post_install", "-at_install")
 class Test(TransactionCase):
     @classmethod
-    def setUpClass(cls, chart_template_ref=None):
+    def setUpClass(cls):
         super().setUpClass()
+
+        filters_obj = cls.env["ir.filters"]
+        filters_form = Form(filters_obj)
+        filters_form.name = "Test No Search group"
+        filters_form.type = "search"
+        filters_form.model_id = "ir.filters.group"
+        filters_form.search_field_id = cls.env.ref(
+            "base_custom_filter.field_ir_filters_group__display_name"
+        )
+        filters_form.filter_domain = "['display_name', 'ilike', self]"
+        filters_form.group_ids.add(cls.env.ref("base.group_system"))
+        filters_form = filters_form.save()
```

Comment:
I'd prefer not to have these lines unless it's for the purpose of a "setup", which is currently not the case. IMO, there is no point adding another pattern to confirm that the form can be saved.